.github/workflows/pr-test-rust.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Test
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - "rust/*"
+  pull_request:
+    branches: [ main ]
+    paths:
+      - "rust/*"
+  workflow_dispatch:
+
+concurrency:
+  group: pr-test-${{ github.ref }}
```

Comment:
Use a different name to resolve conflict. Otherwise, it will cancel other workflows.

Reply:
fixed

Reply:
@merrymercy do you know what is the pending "finish" in the ci?